samples/cf-multi-tenant-application/README.md

```diff
@@ -0,0 +1,55 @@
+# SAP Cloud SDK for JavaScript Multi-Tenant Sample Application
+
+## Description
+
+The code sample in this directory is a reference for the [multi-tenant application tutorial](https://sap.github.io/cloud-sdk/docs/js/tutorials/multi-tenant-application).
+You need to make adjustments to the code before deployment.
+The terms you need to replace are given in all caps and start with _`YOUR_`_ e.g. _`YOUR*REGION`*.
```

Comment:
[q] _`YOUR_`_ e.g. _`YOUR*REGION`*. This looks a bit strange to me. 
Did you mean:  e.g. _`YOUR_*REGION*`_ ?